### PR TITLE
Design create page

### LINKED ISSUE #48: Refactor code, clean up - Create Page
# Value Statement
As a user seeing the create page for the first time, I want to create a flowing form that is responsive and easy to read so that the user can enjoy the experience of filling out their profile details.

# Description

Screenshot 2021-05-12 at 11 16 25

# Acceptance Criteria

- The form should be styled.
- The code for this page should be tidied with components and imported functions used where possible.

# Tasks
- [x] Create a better structured form with `display: grid;`.
- [x] Move large chunks of code to components.
- [x] Where applicable, import functions.
- [x] Refactor where possible.

# Complexity
Medium/Low

### PR DESCRIPTION
**Pull request by hugh-burgess**
- Redesigned the create profie page
- Implemented a working counter for the character count.
- Character count accounts for backspace.
- Grid implemented and better design set.

### CODE REVIEW

File: client/src/components/create-page/CreateProfile.css [hunk 1]
```diff
@@ -1,95 +1,127 @@
-.create-profile-page {
+.background-create-page {
   background-color: var(--light-blue);
```
Comment (bwnstck) [inline client/src/components/create-page/CreateProfile.css]:
```suggestion
  background-color: var(--background-light);
```

"It’s better for the (variable) to reflect what it is than what it looks like."

https://css-tricks.com/what-do-you-name-color-variables/

File: client/src/components/create-page/CreateProfile.css [hunk 2]
```diff
@@ -1,95 +1,127 @@
...
-  max-width: 100vw;
+.image-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  grid-area: 1 /1 /2/2;
+  height: 100%;
+}
+.create-profile-image {
+  border: var(--deep-blue) 3px solid;
```
Comment (bwnstck) [inline client/src/components/create-page/CreateProfile.css]: you could use  a css-variable for the border as well if you reuse it :)

File: client/src/components/create-page/CreateProfile.css [hunk 3]
```diff
@@ -1,95 +1,127 @@
...
 
-.top-wrapper-name-title {
-  width: 100%;
-  text-align: center;
-  font-family: "Cookie", cursive;
-  font-size: 3em;
+.wrapper-name-title {
+  width: 80%;
+  text-align: left;
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (bwnstck) [inline client/src/components/create-page/CreateProfile.css]: same for font-familys

File: client/src/components/create-page/CreateProfile.css [hunk 4]
```diff
@@ -1,95 +1,127 @@
...
+}
+
+.wrapper-name {
+  width: 80%;
+  height: 10%;
   align-self: center;
-  text-align: center;
+  text-align: left;
+  resize: none;
+  box-shadow: 0px 7px 0px 0px rgba(0, 0, 0, 0.32);
```
Comment (bwnstck) [inline client/src/components/create-page/CreateProfile.css]: and box-shadows 🙃

File: client/src/components/create-page/CreateProfile.css [hunk 5]
```diff
@@ -1,95 +1,127 @@
...
+  width: 100%;
+  justify-content: center;
 }
 
 .create-profile-save-button {
-  grid-row: 4/5;
-  grid-column: 1/2;
-  justify-self: center;
-  align-items: center;
   font-size: 40px;
```
Comment (bwnstck) [inline client/src/components/create-page/CreateProfile.css]:
```suggestion
  font-size: 2.5rem;
```
better use `rem` or `em`
https://css-tricks.com/rem-global-em-local/

File: client/src/components/create-page/CreateProfile.js [hunk 6]
```diff
@@ -70,21 +74,39 @@ export default function CreateProfile() {
     setCounter(counter - 1);
     if (e.target.value === "") {
       setCounter(125);
+    } else if (e.nativeEvent.data === null) {
+      console.log("backspace");
```
Comment (bwnstck) [inline client/src/components/create-page/CreateProfile.js]: do the `console.log()`'s have any purpose?

**Review by bwnstck — APPROVED** [review]
lgtm 👩‍🎤🚀
maybe think about variable-naming and some more of them for maybe `border:` and `box-shadow:`

File: client/src/components/create-page/CreateProfile.css (hunk 4 above)
Comment (hugh-burgess) [reply client/src/components/create-page/CreateProfile.css]: excellent ideas, thanks alot!

**Pull request merged**